### PR TITLE
Add entity conflict summary table without tests

### PR DESCRIPTION
**Pull request by matthew-white**

### CODE REVIEW

**Review by ktuite — APPROVED** [review]
This will be exciting to get merged!

**Review by matthew-white — COMMENTED** [review]
Notes from interactive review.

We didn't review the changes to test/.

File: test/components/entity/list.spec.js [hunk 1]
```diff
@@ -231,36 +231,52 @@ describe('EntityList', () => {
...
+        .afterResponse(component => {
+          const modal = component.getComponent(EntityResolve);
+          modal.props().entity.__id.should.equal('e2');
+          return modal.get('.btn-primary').trigger('click');
+        })
+        .request(component => {
+          const button = component.get('.entity-metadata-row:nth-child(2) .resolve-button');
+          return button.trigger('click');
+        })
+        .respondWithData(relevantToConflict)
```
Comment (matthew-white) [inline test/components/entity/list.spec.js:275]: Return different data for the first and second entities.

File: test/components/entity/resolve.spec.js [hunk 2]
```diff
@@ -1,78 +1,141 @@
...
+        .modify(resolve)
+        .complete()
+        .request(async (modal) => {
+          await modal.setProps({ state: false, entity: null });
+          await modal.setProps({
+            state: true,
+            entity: testData.entityOData().value[0]
+          });
+        })
+        .respondWithData(relevantToConflict)
```
Comment (matthew-white) [inline test/components/entity/resolve.spec.js:135]: Return different data for the different entity.

**Review by matthew-white — COMMENTED** [review]
@ktuite and I just reviewed the changes to test/. I still have more tests to write, but I wanted to review the changes that I had made.

We also reviewed some of the changes to src/ from f87db2cd4e7598b75bdfea1120e60b52b46cc5a5.

**Pull request merged**